addons/google_calendar/models/google_calendar.py

```diff
@@ -350,118 +348,112 @@ def get_one_event_synchro(self, cr, uid, google_id, context=None):
 
         url = "/calendar/v3/calendars/%s/events/%s" % ('primary', google_id)
         try:
-            status, content, ask_time = self.pool['google.service']._do_request(cr, uid, url, params, headers, type='GET', context=context)
+            status, content, ask_time = self.env['google.service']._do_request(url, params, headers, type='GET')
         except:
             _logger.info("Calendar Synchro - In except of get_one_event_synchro")
             pass
 
         return status_response(status) and content or False
 
-    def update_to_google(self, cr, uid, oe_event, google_event, context):
-        calendar_event = self.pool['calendar.event']
+    def update_to_google(self, oe_event, google_event):
+        CalendarEvent = self.env['calendar.event']
```

Comment:
***CalendarEvent*** use at once so don't need to occupy memory :) Save some bytes